docs/en/02_Developer_Guides/01_Templates/11_Partial_Template_Caching.md

```diff
@@ -0,0 +1,282 @@
+---
+title: Partial Template Caching
+summary: Cache a section of a template Reduce rendering time with cached templates and understand the limitations of the ViewableData object caching.
+icon: tags
+---
+
+## Partial template caching
+
+Partial template caching is a feature that allows caching of rendered portions of templates. Cached content
+is fetched from a [cache backend](../performance/caching), instead of being regenerated repeatedly.
+
+
+### Base syntax
+
+Here is an example of the basic usage. This is not a definitive example of the syntax, but it shows the most
+common use case.
+
+[note]
+See also [Complete Syntax definition](#complete-syntax-defintition) section
+[/note]
+
+```ss
+<% cached $CacheKey if $Condition %>
+  <!-- Content block -->
+<% end_cached %>
+```
+
+The key parts are `$CacheKey`, `$Condition` and `<!-- Content block -->`.  
+The following sections explain every one of them in more detail.
+
+
+#### $CacheKey
+
+The `$CacheKey` is a string identifying the block content within the cache storage.
+
+The form is `[$key[, $...]]`, where `$key` is any valid template expression and you can have any number of them delimited by comma.
+
+Ultimate value of the cache key includes the following items:
+  - hash of `SilverStripe\View\SSViewer::$global_key` rendered within current template context
```

Comment:
`private static $global_key = '$CurrentReadingMode, $CurrentUser.ID';`

Will that be updated to say "LIVE, 0" at runtime?

Is that automatically applied to the cachekey anyway?  Or do you have to manually add it yourself?  If it's automatic then why are you including it here?

Reply:
yeah, that's automatically applied to every cache key. yes, it's rendered in runtime and becomes "live, 0".

Reply:
yeah I think just remove this whole 'ultimate' section or at least rewrite it all

Reply:
Not sure we can remove it as it explains the basic behaviour of cache keys. Will rewrite it I suppose.